invenio_db/cli.py

```diff
@@ -80,11 +86,19 @@ def create(verbose):
 def drop(verbose):
     """Drop tables."""
     click.secho('Dropping all tables!', fg='red', bold=True)
+    schema_dropper = SchemaDropper(_db.engine.dialect, _db.engine.connect())
     with click.progressbar(reversed(_db.metadata.sorted_tables)) as bar:
         for table in bar:
             if verbose:
                 click.echo(' Dropping table {0}'.format(table))
-            table.drop(bind=_db.engine, checkfirst=True)
+            try:
+                table.drop(bind=_db.engine, checkfirst=True)
+            except InternalError:
```

Comment:
I had to add the try except  because `table.drop(bind=_db.engine, checkfirst=True)` doesn't work after `alembic upgrade heads` and `schema_dropper.connection.execute(DropTable(table))` doesn't work after `db create`